index.html

```diff
@@ -4,6 +4,39 @@
     <title>The Best Recipes!</title>
   </head>
   <body>
-    <h1>Recipes</h1>
+    <h1>Recipes - Selected Japanese and Italian Cuisines</h1>
+    <br>
+    <div>
+      <h2>Japanese Cuisine Recipes</h2>
+    </div>
+    <br>
+    <div> 
+      <ul>
+        <li><a href = "katsudon.html" target="_blank">Katsu Don</a></li>
```

Comment:
Just for good practice, if using target blank, we also want to include this: `rel="noopener noreferrer"`

Reason: https://www.jitbit.com/alexblog/256-targetblank---the-most-underestimated-vulnerability-ever/